package.json

```diff
@@ -95,6 +95,7 @@
     "@types/jasmine": "~2.8.6",
     "@types/jasminewd2": "~2.0.3",
     "@types/jest": "^23.0.0",
+    "ngx-toastr": "^10.2.0",
```

Comment:
Can you also add the package name to the peer dependencies of the package library?